pkgs/by-name/to/tomboy-ng/package.nix

```diff
@@ -0,0 +1,79 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fpc,
+  lazarus,
+  autoPatchelfHook,
+
+  glib,
+  cairo,
+  pango,
+  gtk2,
+  gdk-pixbuf,
+  at-spi2-atk,
+  xorg,
+  libnotify,
+
+  nix-update-script,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tomboy-ng";
+  version = "0.39";
+
+  src = fetchFromGitHub {
+    owner = "tomboy-notes";
+    repo = "tomboy-ng";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-2zU61/ulFJa1337SqbUQFi/Hv5s+wM3h3t/nIR/3XvM=";
+  };
+  kcontrols = fetchFromGitHub {
```

Comment:
Please move this into a let in

Reply:
Wouldn't that make it difficult to override this source derivation?